relay/kusama/src/lib.rs

```diff
@@ -684,6 +683,7 @@ impl pallet_staking::Config for Runtime {
 	type NominationsQuota = pallet_staking::FixedNominationsQuota<{ MaxNominations::get() }>;
 	type MaxUnlockingChunks = frame_support::traits::ConstU32<32>;
 	type HistoryDepth = frame_support::traits::ConstU32<84>;
+	type MaxControllersInDeprecationBatch = ConstU32<5169>;
```

Comment:
quite big, how we settled on this number?

Reply:
please see @rossbulat's [comment](https://github.com/polkadot-fellows/runtimes/pull/159#discussion_r1492649084)